scrapi/base/helpers.py

```diff
@@ -135,7 +135,10 @@ def oai_process_uris(*args):
                 provider_uris.append(found_url)
 
     try:
-        canonical_uri = (provider_uris + object_uris)[0]
+        if kwargs.get('use_doi') and len(object_uris) > 0:
```

Comment:
Would be better to make this a top-level named argument, rather than put it in kwargs.